docs/transfer/1-3-transfer.puml

```diff
@@ -75,21 +75,32 @@ rnote left of D3 #Light
 end note
 D3 -> S: ""PUT /transfers/321""
 S --> D3: ""200 OK""
+D3 -> CB: You have received funds from Ayeesha!
 S -> D2: ""PUT /transfers/321""
 D2 --> S: ""200 OK""
 
 
-rnote over S #Light
-**""PATCH /thirdpartyRequests/transactions/123""**
-""FSPIOP-Source: switch""
+D2 -> D2: Look up ThirdpartyTransactionRequestId from the transferId
```

Comment:
there should be the corresponding action to store `transferId-ThirdpartyTransactionRequestId` pair somewhere

Reply:
Agreed